providers/Bandcamp/mod.ts

```diff
@@ -213,6 +217,19 @@ export class BandcampReleaseLookup extends ReleaseLookup<BandcampProvider, Album
 		}
 		const tracklist = tracks.map(this.convertRawTrack.bind(this));
 
+		if (current.type === 'track') {
+			const track = tracklist[0];
+			track.number = 1;
```

Comment:
In my updated JSON types we have `TrackInfo.track_num: number | null`, meaning the `null` case has to be handled in `convertRawTrack` anyway, where we can add an `index` parameter and use it as a fallback.
That solution also has the benefit that it still grabs the correct track number if someone looks up an album track (if we decide to allow that, see my other comment).

Reply:
Removed that too. It now shows a blank track number for standalone tracks, but is indeed filled in correctly for album tracks, as you've noted.

Reply:
My stashed changes would use tracklist index + 1 for tracks without number (only affects standalone track as far as I have seen).
Let me know when you are ready, then I will apply and commit my JSON type changes directly to your branch.

P.S. Can you prefix your commit messages with `feat(Bandcamp): ` please, one day I will use them to generate changelogs.

Reply:
Should be ready now.